src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -100,6 +115,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     on the TargetFramework.
   -->
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.NuGetOfflineCache.targets" />
+
+  <!-- Disable some implicit behaviors for "classic" projects which aren't using Microsoft.NET.Sdk -->
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
+    <!-- Don't disable implicit framework references for .NET Core or .NET Standard, in that case it controls the reference to the
+         "targeting pack" package.  Do disable them for .NET Framework, because in that case it disables the Reference items which
+         will be in the project file for "classic" projects. -->
```

Comment:
This also isn't necessary for C++ .net core case. I am wondering if we should reduce this down to the things that we really need for C++. We should discuss with @dsplaisted

Reply:
pending a meeting with Daniel

Reply:
What I mean here is that this is doing special work for targeting net framework, but that is not something we will be supporting for the C++ feature. We should check what happens now if you put TargetFramework=net46 in vcxproj. I think it will break as the features that are importing SDK in VC project targets are doing so along with netcore specific compiler switches, etc.?

Reply:
remove it for now. and have better error message when there is fullframework tfm

Reply:
Removed. And added check for fullframework